.github/workflows/go-release.yml

```
@@ -42,14 +42,14 @@ jobs:
       - name: Pull main image
         id: pull-main-image
         run: |
-          echo "Pulling image: ${{ env.REGISTRY }}/${{ env.ORGANIZATION }}/adapters:main-${{ steps.short-sha.outputs.sha }}"
-          docker pull ${{ env.REGISTRY }}/${{ env.ORGANIZATION }}/adapters:main-${{ steps.short-sha.outputs.sha }}
+          echo "Pulling image: ${{ env.REGISTRY }}/${{ env.ORGANIZATION }}/${{ github.event.repository.name }}:main-${{ steps.short-sha.outputs.sha }}"
```

WDYT about using this and completely removing the custom `ORGANIZATION` env? This should also allow forked repos to work properly without having to modify this env. ([ref](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context:~:text=or%20tag.-,github.repository,-string))
```suggestion
          echo "Pulling image: ${{ env.REGISTRY }}/${{ github.repository }}:main-${{ steps.short-sha.outputs.sha }}"
```

We should also keep this consistent with `.github/workflows/go.yml`

---

Yeah, sounds good.